src/hotspot/share/oops/arrayOop.hpp

```diff
@@ -71,10 +57,29 @@ class arrayOopDesc : public oopDesc {
   // aligned 0 mod 8.  The typeArrayOop itself must be aligned at least this
   // strongly.
   static bool element_type_should_be_aligned(BasicType type) {
```

Comment:
The comment above makes it sound like this is only valid for `typeArrayOop`. Should be updated to reflect the change.

Reply:
Ok.